include/seqan/platform.h

```diff
@@ -58,30 +52,12 @@
 // SEQAN_FORWARD .......... pass a variable as (of type T) as it was given to a function
 // SEQAN_MOVE ............. pass a variable to a function and never use it again
 
-#ifdef SEQAN_CXX11_STANDARD
-
-    #define SEQAN_AUTO_PTR_NAME     unique_ptr
-    #define SEQAN_FORWARD_ARG       &&
-    #define SEQAN_FORWARD_CARG      &&
-    #define SEQAN_FORWARD_RETURN    &&
-    #define SEQAN_FORWARD(T, x)     std::forward<T>(x)
-    #define SEQAN_MOVE(x)           std::move(x)
-
-#else  // #ifdef SEQAN_CXX11_STANDARD
-
-    #define SEQAN_AUTO_PTR_NAME     auto_ptr
-    #define SEQAN_FORWARD_ARG       &
-    #define SEQAN_FORWARD_CARG      const &
-    #define SEQAN_FORWARD_RETURN
-    #define SEQAN_FORWARD(T, x)     x
-    #define SEQAN_MOVE(x)           x
-
-#endif  // #ifdef SEQAN_CXX11_STANDARD
-
-// Is the C++11 STL (thread, atomic, chrono) available?
-#if defined(SEQAN_CXX11_STANDARD) && (!defined(_MSC_VER) || _MSC_VER >= 1700) && !defined(PLATFORM_WINDOWS_MINGW)
-#define SEQAN_CXX11_STL
-#endif
+#define SEQAN_AUTO_PTR_NAME     unique_ptr
+#define SEQAN_FORWARD_ARG       &&
+#define SEQAN_FORWARD_CARG      &&
+#define SEQAN_FORWARD_RETURN    &&
+#define SEQAN_FORWARD(T, x)     std::forward<T>(x)
+#define SEQAN_MOVE(x)           std::move(x)
 
```

Comment:
- please do replace these macros in the library with their spedific RHS, except for std::forward this should be an easy search and replace!
  - but leave them defined anyway in case external apps use the macros
  - add a comment to the source code that declares the use of the macros as deprecated